dm/tests/tls/run.sh

```diff
@@ -118,30 +118,31 @@ function test_worker_download_certs_from_master() {
 		"start-task $WORK_DIR/dm-task.yaml"
 
 	# task should be paused.
-	run_dm_ctl_with_tls_and_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" $cur/conf/ca.pem $cur/conf/dm.pem $cur/conf/dm.key \
-		"query-status test" \
-		"\"result\": true" 2 \
-		"\"stage\": \"Paused\"" 1
-
-	# change task-ca.pem name to test wheather dm-worker will dump certs from dm-master
-	mv "$cur/conf/task-ca.pem" "$cur/conf/task-ca.pem.bak"
-
-	# kill dm-worker 1 and clean the failpoint
-	export GO_FAILPOINTS=''
-	ps aux | grep dm-worker1 | awk '{print $2}' | xargs kill || true
-	check_port_offline $WORKER1_PORT 20
-	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $WORK_DIR/dm-worker1.toml
-	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT "$cur/conf/ca.pem" "$cur/conf/dm.pem" "$cur/conf/dm.key"
-
-	# dm-worker will dump task-ca.pem from dm-master and save it to dm-worker-dir/ca.pem
-	# let's try use this file to connect dm-master
-	run_dm_ctl_with_tls_and_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" /tmp/dm_test/tls/worker1/ca.pem $cur/conf/dm.pem $cur/conf/dm.key \
-		"query-status test" \
-		"\"result\": true" 2
-
-	echo "check data"
-	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
-	mv "$cur/conf/task-ca.pem.bak" "$cur/conf/task-ca.pem"
+	# TODO: revert this when lightning support tls
```

Comment:
you can comment the whole test  `test_worker_ha_when_enable_source_tls` 😂

Reply:
hope you can finish it soon so we'll revert it😂